commons/src/test/java/io/hyscale/commons/config/TestSetupConfigForAbsolutePath.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.config;
+
+import io.hyscale.commons.constants.ToolConstants;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.File;
+import java.net.URL;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TestSetupConfigForAbsolutePath {
+    private  static StringBuilder stringBuilder = new StringBuilder();
+    private static String absolutePath = stringBuilder.append(ToolConstants.FILE_SEPARATOR).append("absolute").append(ToolConstants.FILE_SEPARATOR).append("path").append(ToolConstants.FILE_SEPARATOR).append("test").toString();
```

Comment:
Since you are setting absolute path in @BeforeEach , do the build also there it is more relevant of handling the build rather than keeping it global variable .